internal/controlplane/handlers_user.go

```diff
@@ -199,11 +199,21 @@ func (s *Server) getUserDependencies(ctx context.Context, user db.User) ([]*pb.P
 			return nil, err
 		}
 
+		// Try to parse the project metadata to complete the response fields
+		var pDisplay, pDescr string
+		meta, err := projects.ParseMetadata(&pinfo)
+		if err == nil {
+			pDisplay = meta.Public.DisplayName
+			pDescr = meta.Public.Description
+		}
+
 		projectsPB = append(projectsPB, &pb.Project{
-			ProjectId: proj.String(),
-			Name:      pinfo.Name,
-			CreatedAt: timestamppb.New(pinfo.CreatedAt),
-			UpdatedAt: timestamppb.New(pinfo.UpdatedAt),
+			ProjectId:   proj.String(),
+			Name:        pinfo.Name,
+			CreatedAt:   timestamppb.New(pinfo.CreatedAt),
+			UpdatedAt:   timestamppb.New(pinfo.UpdatedAt),
+			DisplayName: pDisplay,
+			Description: pDescr,
```

Comment:
@kantord - hey, in case the parsing above fails for some reason is it okay with you to have the default variable values here, i.e. empty strings?

Reply:
you mean specifically for description? I think for description, empty string is a reasonable fallback value, but for displayName I think that using the name as a fallback value would make more sense

Reply:
also, I'm not very informed about how the BE codebase works, but as I understand the fallback logic happens at insertion time, meaning this code itself does not address existing data in the database. I guess those could be fixed using a database migration or a manual query?

Reply:
I'm pretty sure this code is happening on the way _out_ of the database when answering the query, not on insertion.  (That's both what I would expect, and matches the pattern in the code where we're constructing a protocol buffer for output.)  If we change 203 to:

```golang
pDisplay := pinfo.Name
pDescr := ""
```

before calling `ParseMetadata`, that should be sufficient to fill in the defaults when the metadata doesn't exist in the database, without needing a backfill.

Reply:
Thanks for the suggestion @evankanderson , I've changed the initial values as suggested :)